doc/release/technotes/README.auxIO.rst

```diff
@@ -12,8 +14,13 @@ systems simultaneously.
 Dependencies
 ------------
 
-For each AIO system described here, there is a corresponding README.<system> in
-this directory detailing how to install and use them.
+Each AIO system is typically described in the support module for it.
+See:
+
+ * :mod:`HDFS`
```

Comment:
What is `:mod:` doing and do you intend for these to be blue links? (They are in Github render)